server/auth-router.js

```diff
@@ -135,6 +135,11 @@ export default function authRouter(config = {}) {
 			options.login_hint = 'signUp';
 		}
 
+		// Guest Username
+		if (req.query.username) {
+			options.username = req.query.username;
```

Comment:
Should we be adding this to the `login_hint` JSON?

Reply:
i think `login_hint` has a different purpose, so adding an extra param for `username` is good to clarifying this new purpose.